index.js

```diff
@@ -24,12 +24,31 @@ try {
   const logoPosition = core.getInput('logoPosition');
   const style        = core.getInput('style');
   const cacheSeconds = core.getInput('cacheSeconds');
+  const valColorRange = core.getInput('valColorRange');
+  const minColorRange = core.getInput('minColorRange');
+  const maxColorRange = core.getInput('maxColorRange');
+  const invertColorRange = core.getInput('invertColorRange');
 
   if (labelColor != '') {
     content.labelColor = labelColor;
   }
 
-  if (color != '') {
+  if (minColorRange != '' && maxColorRange != '' && valColorRange != '') {
+    var max = parseFloat(maxColorRange);
+    var min = parseFloat(minColorRange);
+    var val = parseFloat(valColorRange);
+    if (val < min) val = min;
+    if (val > max) val = max;
+    var hue = 0
```

Comment:
I think this local variable could be declared with `let`:

```suggestion
    let hue = 0
```

Reply:
Also agree